src/openrct2/peep/Staff.cpp

```diff
@@ -643,7 +643,10 @@ bool Staff::DoHandymanPathFinding()
                 bool chooseRandom = true;
                 if (litterDirection != INVALID_DIRECTION && pathDirections & (1 << litterDirection))
                 {
-                    if ((scenario_rand() & 0xFFFF) >= 0x1999)
+                    /// When in a queue path make the probability of following litter much lower
+                    /// as handymen often get stuck when there is litter on a normal path next to a queue they are in
+                    uint32_t chooseRandomProbability = pathElement->IsQueue() ? (0xFFFF - 0x1999) : 0x1999;
```

Comment:
Also just use `0xE666` for the value. I've a branch that is refactoring these constants to be easier to understand. (0x1999 means 10%)

Reply:
Alright I will use `0XE666` for now